.github/CODEOWNERS

```diff
@@ -20,3 +20,4 @@
 /traffic_director/      @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /vertex_ai/             @terraform-google-modules/dee-data-ai @terraform-google-modules/terraform-samples-reviewers
 /vpc/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
+/integrationconnectors/ @googlecloudplatform/connectors-eng-team @terraform-google-modules/terraform-samples-reviewers
```

Comment:
Still gives the same error, please suggest.

Reply:
For this PR, please remove this line.

Use a new PR to update code-owners, after creating @terraform-google-modules/connectors-eng-team group.